### ISSUE TITLE
🐛 BUG: 

### ISSUE THREAD

**Issue by nneil** [cloudflare/workers-sdk#5662]
### Which Cloudflare product(s) does this pertain to?

Wrangler core

### What version(s) of the tool(s) are you using?

3.51.2

### What version of Node are you using?

v20.12.1

### What operating system and version are you using?

Windows 10

### Describe the Bug

When I execute an SQL file on my newly created local D1 database I get a "not authorized" error.

```
PS C:\Projects\cloudflare> wrangler d1 create dev
 ⛅️ wrangler 3.51.2

[[d1_databases]]
binding = "DB" # i.e. available in your Worker on env.DB
database_name = "dev"
database_id = "77922c77-3436-431c-ac2d-4114a75840da"
PS C:\Projects\cloudflare> d1 execute dev --file .\sql\db_schema-drop.sql   
 ⛅️ wrangler 3.51.2
-------------------
🌀 Mapping SQL input into an array of statements
🌀 Executing on local database dev (77922c77-3436-431c-ac2d-4114a75840da) from .wrangler\state\v3\d1:
🌀 To execute on your remote database, add a --remote flag to your wrangler command.

X [ERROR] not authorized
```

**Comment by luckykellan** [comment]
I am experiencing the same issue as reported in this thread. Here are some details about my environment and the error I am encountering:

- **Wrangler Version**: 3.60.1

- **Operating System**: mac 

Has there been any progress on this issue, or does anyone have a workaround or solution? Any help would be greatly appreciated.

**Comment by luckykellan** [comment]
I've managed to resolve the issue that was discussed in this thread. Here's what worked for me:

For anyone experiencing similar issues, please recheck your SQL file to ensure that the `BEGIN TRANSACTION` and `COMMIT;` statements have been removed, as mentioned in the [documentation guidelines](https://developers.cloudflare.com/d1/build-with-d1/import-data/#convert-sqlite-database-files). This specific adjustment solved the problem for me.

Hope this helps anyone else facing the same issue.

**Comment by penalosa** [comment]
@nneil did @luckykellan's suggestion solve this for you?

**Comment by nneil** [comment]
Recently the ability to easily send email from Pages, using Mailchannel, was discontinued. I saw a message about this in the console, but it was not readily apparent to me what the replacement was. I wasted a day trying to find a replacement myself and was unable to find anything reasonable. In one case I was blocked because "my website does not have enough content". In another I could not jump through their hoops.

This made me realise that I cannot afford to rely on any third-parties. 

Until Cloudflare provides it's own email service, as AWS does, I have been forced to go back to traditional development on normal cloud servers. Consequently I have not done any more work on this issue.
